communications/ff_msgs/msg/CpuStateStamped.msg

```diff
@@ -45,3 +45,5 @@ float32 temp
 # or not all of them are enabled.
 ff_msgs/CpuState[] cpus
 
+# Load usage of individual ROS nodes
+ff_msgs/CpuNodeState[] load_nodes
```

Comment:
I would highly recommend we avoid lightly changing message schemas, because of the problems we discussed before with losing the ability to load data from older bags.

In this case with adding a new field, I would recommend either:
1. Leave the old message schema alone and publish the new field as a separate new message. Old bags would have only the old message and new bags would have both the old and new messages.
2. Leave the old message schema alone and define a new extended message schema as you have it here, but make sure both the new message schema name and the topic it is published on are different (e.g. CpuStateStamped2). Old bags would have only the old message and new bags would have only the new extended message. Analysis tools could still load old bags because the old message definition would still be present, although deprecated for future use.

In the rare cases when you think it's worthwhile to propose changing a message schema, I think we should have a rule to highlight that clearly in the PR summary so it can't accidentally slip through the review process.

Reply:
I think option 2 might be better because option 1 would imply updating all the recording profiles

Reply:
For this specific case I confirmed with Brian that the message was ok to change and Katie confirmed that the bridge is not altered.
The performance script that is a part of the PR that analyses the message will work with both old and new definitions given that we added to the message (uses python) instead of modifying it.

Reply:
It sounds like you're saying that the Python bag reader is able to read messages from old bags where the messages on the topic of interest have a missing field according to the current (updated) message schema. That's really surprising. (How does it fill in the value of the missing field?)

Really glad to hear that works!